### PR TITLE
Eliminate PsmCount from PEP calculation

### PR DESCRIPTION
**Pull request by trishorts**
cuz some smart people think that introduces bias.

https://pubmed.ncbi.nlm.nih.gov/21391616/

image

### CODE REVIEW

File: MetaMorpheus/EngineLayer/PsmTsv/PsmTsvWriter.cs [hunk 1]
```diff
@@ -95,20 +94,6 @@ internal static (string ResolvedString, Dictionary<string, int> ResolvedValue) R
             }
         }
 
-        internal static (string ResolvedString, double? ResolvedValue) ResolveF2(IEnumerable<double> enumerable)
```
Comment (nbollis) [inline MetaMorpheus/EngineLayer/PsmTsv/PsmTsvWriter.cs:98]: Why did this one go?

File: MetaMorpheus/EngineLayer/PeptideSpectralMatch.cs [hunk 2]
```diff
@@ -189,6 +195,14 @@ public void SetFdrValues(double cumulativeTarget, double cumulativeDecoy, double
             };
         }
 
+        public void SetQandPEPvalues(double qValue, double qValueNotch, double pep, double pepQValue)
```
Comment (nbollis) [inline MetaMorpheus/EngineLayer/PeptideSpectralMatch.cs:198]: Will the other fields in FdrInfo need to be updated as well?

File: MetaMorpheus/EngineLayer/PsmTsv/PsmTsvWriter.cs (hunk 1 above)
Comment (trishorts) [reply MetaMorpheus/EngineLayer/PsmTsv/PsmTsvWriter.cs:98]: no longer needed.

File: MetaMorpheus/EngineLayer/PeptideSpectralMatch.cs (hunk 2 above)
Comment (trishorts) [reply MetaMorpheus/EngineLayer/PeptideSpectralMatch.cs:207]: see comment in revised PR

**Review by nbollis — APPROVED** [review]

**Review by Alexander-Sol — APPROVED** [review]

**Review by elaboy — APPROVED** [review]

**Pull request merged**